### ISSUE TITLE
update to 10.0.8 failed, the server not accessible

### ISSUE THREAD

**Issue by ken948** [owncloud/core#31300]
My owncloud server is not accessible!!!

 
### Steps to reproduce
1.  yum update
2.
3.

### Actual behaviour
Tell us what should happen
Repair warning: Exception
Repair warning: App directory already exists
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps

### Server configuration
Centos 7

**Web server:**
ame        : httpd
Arch        : x86_64
Version     : 2.4.6
Release     : 67.el7.centos.6

mysql  Ver 14.14 Distrib 5.6.40, for Linux (x86_64)

PHP 7.0.29

not accessible

**Updated from an older ownCloud or fresh install:**
update from previous 10.0.8

**Where did you install ownCloud from:**
Don't understand the question

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results into https://gist.github.com/ and puth the link here.
```

**The content of config/config.php:**

Log in to the web-UI with an administrator account and click on
'admin' -> 'Generate Config Report' -> 'Download ownCloud config report'
This report includes the config.php settings, the list of activated apps
and other details in a well sanitized form.

or 

If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

*ATTENTION:* Do not post your config.php file in public as is. Please use one of the above
methods whenever possible. Both, the generated reports from the web-ui and from occ config:list
consistently remove sensitive data. You still may want to review the report before sending.
If done manually then it is critical for your own privacy to dilligently
remove *all* host names, passwords, usernames, salts and other credentials before posting.
You should assume that attackers find such information and will use them against your systems.
root@owncloud owncloud]# sudo -u apache php occ config:list system
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
{
    "system": {
        "updatechecker": false,
        "instanceid": "ocid0jawwgfk",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.42.3",
            "owncloud.perfectsystems.ca"
        ],
        "datadirectory": "\/var\/www\/html\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/192.168.42.3\/owncloud",
        "dbtype": "mysql",
        "version": "10.0.7.2",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "theme": "",
        "maintenance": false,
        "loglevel": 2
    }
}

**List of activated apps:**
[
`
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder.
root@owncloud owncloud]# sudo -u apache php occ app:list
ownCloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Enabled:
  - comments: 0.3.0
  - configreport: 0.1.1
  - dav: 0.3.2
  - federatedfilesharing: 0.3.1
  - federation: 0.1.0
  - files: 1.5.1
  - files_external: 0.7.1
  - files_pdfviewer: 0.8.2
  - files_sharing: 0.10.1
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - market: 0.2.4
  - notifications: 0.3.2
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - updatenotification: 0.2.1
Disabled:
  - encryption
  - external
  - theme-example
  - user_external

**Are you using external storage, if yes which one:** local/smb/sftp/...

No..

**Are you using encryption:** yes/no

No..

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No..

### Client configuration
**Browser:**
Google Chrome

**Operating system:**
Fedora server 27
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
Preparing update
Set log level to debug
Repair warning: Exception
Repair warning: App directory already exists
Repair warning: You have incompatible or missing apps enabled that could not be found or updated via the marketplace.
Repair warning: Please install or update the following apps manually or disable them with: occ app:disable templateeditor
Repair warning: For manually updating, see https://doc.owncloud.org/server/10.0/go.php?to=admin-marketplace-apps
OC\RepairException: Upgrade is not possible

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @kdslkdsaldsal.

Possibly related issues are https://github.com/owncloud/core/issues/14905 (Update 8.0 -> 8.0.2 fails due to lack of web server write access), https://github.com/owncloud/core/issues/16820 (Upgrade to 8.0.4 fails), https://github.com/owncloud/core/issues/16381 (update to 8.0.3 fails), https://github.com/owncloud/core/issues/16280 (Update to 8.0.3 failed), and https://github.com/owncloud/core/issues/29143 (failed upgrade to 8.0.16).

**Comment by ken948** [comment]
I followed the warning message and disabled the app "templateeditor".  **Please install or update the following apps manually or disable them with: occ app:disable templateeditor**
The update started to run and completed  successfully.  
Question:  What does the add "templeteeditor" do?  Do I need it on my owncloud server ?

**Comment by patrickjahns** [comment]
@ken948 

templateeditor was prior to 10.0.8 shipped with ownCloud - with 10.0.8 - we removed it as a default shipped app. Should you require the template editor, it is now freely available on our marketplace: https://marketplace.owncloud.com/apps/templateeditor. 

Templateeditor app itself is not required to run ownCloud - if you haven't used it before - you are fine by just disabling an then upgrading. Should you want to use it at any given point in time - just download the app from the marketplace and install it manually.

I will close the ticket, as the initial issue is resolved
